flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -553,6 +553,45 @@ def set_python_executable(self, python_exec: str):
             .getEnvironmentConfig(self._j_stream_execution_environment)
         env_config.setString(jvm.PythonOptions.PYTHON_EXECUTABLE.key(), python_exec)
 
+    def add_jars(self, *jars_path: List[str]):
+        """
+        Adds a list of jar files that will be uploaded to the cluster and referenced by the job.
+
+        :param jars_path: Path of jars that delimited by ';'.
+        """
+        add_jars_to_context_class_loader(*jars_path)
+        jvm = get_gateway().jvm
+        jars_key = jvm.org.apache.flink.configuration.PipelineOptions.JARS.key()
+        env_config = jvm.org.apache.flink.python.util.PythonConfigUtil \
+            .getEnvironmentConfig(self._j_stream_execution_environment)
+        set_jar_paths = env_config.getString(jars_key, None)
```

Comment:
Why not as following?
```suggestion
        set_jar_paths = env_config.getString(jars_key, '')
```